nativelib/src/main/resources/scala-native/gc/commix/datastructures/GreyPacket.c

```diff
@@ -105,8 +105,8 @@ GreyPacket *GreyList_Pop(GreyList *list, word_t *greyPacketsStart) {
         if (next.atom != 0) {
             nextValue.atom = next.atom;
         } else {
-            nextValue.sep.idx = headIdx + 1;
-            nextValue.sep.size = head.sep.size - 1;
+            nextValue.sep.idx = UInt24_plus(headIdx, 1);
+            nextValue.sep.size = UInt24_plus(head.sep.size, -1);
```

Comment:
`UInt24_plus` takes a `uint32_t` as second argument, but here we pass a negative number. Should be an `int32_t` instead?

Reply:
Yes, you're right. I've fixed that.